examples/headsail-bsp/src/mmap.rs

```diff
@@ -143,3 +143,6 @@ pub const TIMER0_ADDR: usize = 0x5_0000;
 pub const TIMER1_ADDR: usize = 0x5_0010;
 pub const TIMER2_ADDR: usize = 0x5_0020;
 pub const TIMER3_ADDR: usize = 0x5_0030;
+
+// Base addres for SDRAM configuration registers
+pub const SDRAM_CONFIG_ADDR: usize = 0xFFD00000;
```

Comment:
Ideally you separate the words with an underscore for readability.

Reply:
Underscore added